src/NativeScript/TNSRuntime.mm

```diff
@@ -42,6 +42,7 @@ + (void)initializeMetadata:(void*)metadataPtr {
 
 - (instancetype)initWithApplicationPath:(NSString*)applicationPath {
     if (self = [super init]) {
+        JSC::Options::useJIT() = false;
```

Comment:
Maybe we should move this line of code to the static constructor of the runtime, if it's affecting all instances of the runtime.